### PR TITLE
DevTools > Network reference refresh

### PR DESCRIPTION
**Pull request by sofiayem**
The Network reference is the most viewed page in the DevTools doc set. It had screenshots from M58.

- Recaptured all old screenshots.
- Added cosmetic changes to better comply with https://developers.google.com/style.
- Documented along the way:
   - https://developer.chrome.com/blog/new-in-devtools-96/#payload
   - https://developer.chrome.com/blog/new-in-devtools-94/#nvert-network-filter
   - https://developer.chrome.com/blog/new-in-devtools-92/#wasm

### CODE REVIEW

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/network/reference/index.md [hunk 1]
```diff
@@ -437,54 +394,48 @@ To view HTTP header data about a request:
...
-### View query string parameters {: #query-string }
+- The network resource isn't valid. For example, execute `fetch("https://jec.fyi.com/unknown-url/")` in the **Console**.
+  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/xMKUHQxXLiRDiiUTol5J.png", alt="Provisional headers warning message.", width="800", height="517" %}
 
-To view the query string parameters of a URL in a human-readable format:
+DevTools can also display only provisional headers due to security reasons.
 
-1.  Open the **Headers** tab for the request you're interested in. See [View HTTP headers][16].
-2.  Go to the **Query String Parameters** section.
+### View query string parameters (request payload) {: #query-string }
```
Comment (jecfish) [inline site/en/docs/devtools/network/reference/index.md:420]: There are two types of payload Query str params and Form Data. Probablt change the title to payload or add "and form data".

To debug, login Twitter, open network panel, refresh the page, filter request by fetch/xhr, and go through the list. There are both query str and form data example.

File: site/en/docs/devtools/network/reference/index.md [hunk 2]
```diff
@@ -437,54 +394,48 @@ To view HTTP header data about a request:
...
 
-#### View URL-encoded query string parameters {: #query-string-encodings }
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/1o1ukljKJNORmbulgXiM.png", alt="The source string of the request payload.", width="800", height="418" %}
 
-To view query string parameters in a human-readable format, but with encodings preserved:
+#### View URL-decoded arguments of query string parameters {: #query-string-encodings }
 
-1.  Go to the Query String Parameters section. See [View query string parameters][18].
-2.  Click **view URL encoded**.
+To disable URL-encoding for arguments, on the **Payload** tab, click **decoded**.
```
Comment (jecfish) [inline site/en/docs/devtools/network/reference/index.md:436]:
```suggestion
To disable URL-encoding for arguments, on the **Payload** tab, click **view decoded / view URL-encoded**.
```

Please confirm.

**Review by jecfish — CHANGES_REQUESTED** [review]
Such a big refresh! 👏

File: site/en/docs/devtools/network/reference/index.md [hunk 3]
```diff
@@ -560,26 +501,22 @@ Here's more information about each of the phases you may see in the Timing tab:
 To view the initiators and dependencies of a request, hold Shift and hover over the request in the
```
[the remark this answers is not in the document]
Comment (sofiayem) [reply site/en/docs/devtools/network/reference/index.md]: This is on my todo list for M80.

**Pull request merged**